src/PSRule/Configuration/ExecutionOption.cs

```diff
@@ -50,7 +54,8 @@ public bool Equals(ExecutionOption other)
             return other != null &&
                 LanguageMode == other.LanguageMode &&
                 InconclusiveWarning == other.InconclusiveWarning &&
-                InconclusiveWarning == other.InconclusiveWarning;
+                NotProcessedWarning == other.NotProcessedWarning &&
```

Comment:
Fixed a typo where there was double checks for `InconclusiveWarning`, and added in `NotProcessedWarning` check instead.

Reply:
@ArmaanMcleod Good spot.